.github/workflows/labeler.yml

```diff
@@ -0,0 +1,25 @@
+name: Labeler
+
+on:
+  push:
+    branches:
+      - main
+      - master
+
+jobs:
+  labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repository
+        uses: actions/checkout@v2.3.3
+
+      - name: Run Labeler
+        if: success()
+        uses: crazy-max/ghaction-github-labeler@v3.1.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          yaml-file: .github/labels.yml
+          skip-delete: true
+          dry-run: false
+          exclude: |
+            hacktoberfest*
```

Comment:
```suggestion
```

Let's not include this. The list of "seasonal" labels is open ended, and anyway this is not so important because we skip deletes. I understand it's meant as an example, but I'd prefer to keep examples out of the code base to minimize noise.